common-content/en/module/js1/comparison/index.md

```diff
@@ -28,13 +28,11 @@ Whenever we call `formatAs12HourClock` we expect it to return a particular value
 However, the **current output** is what the function _actually_ returns right now.
 
 👉🏼 [Work through the exercises](#current-1)
-
-{{</tab>}}
-{{<tab name="Exercise 1">}}
+===[[Exercise 1]]
```

Comment:
This is looking pretty broken? https://deploy-preview-1268--cyf-programming.netlify.app/structuring-data/sprints/2/prep/#comparison

<img width="992" alt="Screenshot 2024-12-30 at 11 30 07" src="https://github.com/user-attachments/assets/651031f5-c338-41e9-b72e-ba129a0110ec" />

Reply:
Oh that was my working test block, dammit, so it still has old code in. Sigh! Thanks!

Reply:
No I've broken it somehow in the night! Just by sleeping! This is why sleep is for fools.